### PR TITLE
Avoid using JAXB just for hex conversion.

### PR DESCRIPTION
**Pull request by jasonborg**

### CODE REVIEW

File: src/agent/internals/src/main/java/com/google/devtools/cdbg/debuglets/java/UniquifierComputer.java [hunk 1]
```diff
@@ -58,9 +57,18 @@ public UniquifierComputer(String initializationVector, SortedSet<String> applica
...
   public String getUniquifier() {
-    return DatatypeConverter.printHexBinary(hash.digest());
+    byte[] digest = hash.digest();
+    char[] hex = new char[digest.length * 2];
+    for (int i = 0, j = 0; i < digest.length; i++) {
+      byte b = digest[i];
+      hex[j++] = HEX_DIGITS[(b >> 4) & 0xf];
+      hex[j++] = HEX_DIGITS[b & 0xf];
+    }
+    return new String(hex);
```
Comment (mctavish) [inline src/agent/internals/src/main/java/com/google/devtools/cdbg/debuglets/java/UniquifierComputer.java:71]: I would love to have a teeny tiny unit test for the byte[] to char[] conversion.

**Review by mctavish — APPROVED** [review]

File: src/agent/internals/src/main/java/com/google/devtools/cdbg/debuglets/java/UniquifierComputer.java (hunk 1 above)
Comment (jasonborg) [reply src/agent/internals/src/main/java/com/google/devtools/cdbg/debuglets/java/UniquifierComputer.java]: Done.

File: src/agent/internals/src/test/java/com/google/devtools/cdbg/debuglets/java/FirebaseClientTest.java [hunk 2]
```diff
@@ -1,3 +1,19 @@
+/*
```
Comment (mctavish) [inline src/agent/internals/src/test/java/com/google/devtools/cdbg/debuglets/java/FirebaseClientTest.java:1]: Nit: Technically this isn't part of this set of changes.

**Review by mctavish — APPROVED** [review]

File: src/agent/internals/src/test/java/com/google/devtools/cdbg/debuglets/java/FirebaseClientTest.java (hunk 2 above)
Comment (jasonborg) [reply src/agent/internals/src/test/java/com/google/devtools/cdbg/debuglets/java/FirebaseClientTest.java]: Update removed.

**Pull request merged**